module3/assessments/week2-3448.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: page
+title: Week 2 Assessment
+---
+
+[Follow the instructions on this repo! The README with the instructions is inside the Tourism project.](https://github.com/turingschool-examples/Launch_3448)
```

Comment:
I like what you did where you added two readme's to the repo. With the main page repo having the rubric and instructions on how to get to the README students edit. 

I vote that we only have the rubric in one place, in case we want to edit it later then things don't get out of sync. My vote is to have the rubric in the top level of the repo so everything is all in the repo and the launch.turing page is just a redirection. What do you think?

Reply:
That works for me